src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -1629,10 +1637,18 @@ public SolutionChanges GetChanges(Solution oldSolution)
     public OptionSet Options => this.SolutionState.Options;
 
     /// <summary>
-    /// Analyzer references associated with the solution.
+    /// Analyzer references associated with the solution.n
     /// </summary>
     public IReadOnlyList<AnalyzerReference> AnalyzerReferences => this.SolutionState.AnalyzerReferences;
 
+    /// <summary>
+    /// Fallback analyzer config options by language. The set of languages does not need to match the set of languages of projects included in the current solution snapshot
```

Comment:
ok.  so just to check, when people read from this, they always use TryGet, and don't assume they can blindly index by lang?

Reply:
Indeed. For multiple reasons. C#-only solution only has C# options. Or you can set FallbackAnalyzerOptions to arbitrary dictionary. It may include other languages as well.

Reply:
question though: it's possible to have C# projects and have not have a C# entry in this map, right?  because nothing ensures they stay in sync as the *solution* forks (not the workspace.currentsolution updating).  So code should always assume it can't get these options, right?